File: src/03_input_and_type/main.jl
# Course: ES123
# By: Ben Lauwens

using ES123

# Objective: put the message "Hello, $first_name (age $age)" on the screen
# Input: string first_name, integer age
# Output: "Hello, $first_name (age $age)"
# Processing: nihil

print("Please enter your first name: ")
first_name = chomp(readline())
print("Please enter your age: ")
# parse tries to convert the string input to an integer
age = parse(Int, readline())
println("Hello, ", first_name, " (age ", age, ")")

File: src/03_input_and_type/main.py
# Course: ES123
# By: Ben Lauwens

import ES123

# Objective: put the message "Hello, $first_name (age $age)" on the screen
# Input: string first_name, number age
# Output: "Hello, $first_name (age $age)"
# Processing: nihil

first_name = input("Please enter your first name: ")
# int tries to convert the string input to an integer
age = int(input("Please enter your first age: "))
print("Hello, ", first_name, " (age ", age, ")", sep='')
